Menu/Menu.js

```diff
@@ -33,3 +33,52 @@ let menuItems = [
   Step 6: add the menu component to the DOM.
   
 */
+
+
+//component [...]
+const menuButton = document.querySelector('.menu-button');
+/*new update on header*/
+const header = document.querySelector('.header')
+
+//adding the menu component to the DOM
+menuItems.forEach(add => {
+  menuButton.appendChild(menuCreator(add.arr));
```

Comment:
On line 45, it looks like your looping through your nav links and trying to render them on your 'menuButton'. Instead, you will want to create a new space for them. Probably a good spot to create and element. Maybe one can be a container and create another for list items?